docs/kubectl.md

```diff
@@ -66,4 +66,4 @@ kubectl
 * [kubectl update](kubectl_update.md)	 - Update a resource by filename or stdin.
 * [kubectl version](kubectl_version.md)	 - Print the client and server version information.
 
-###### Auto generated by spf13/cobra at 2015-04-27 22:01:46.529528989 +0000 UTC
+###### Auto generated by spf13/cobra at 2015-04-28 20:12:25.256593317 +0000 UTC
```

Comment:
Agreed, although I don't see how to add it without doing this otherwise I can't seem to commit.